drf_spectacular/openapi.py

```diff
@@ -818,7 +819,7 @@ def _get_response_bodies(self):
 
         if is_serializer(response_serializers) or is_basic_type(response_serializers):
             if self.method == 'DELETE':
-                return {'204': {'description': 'No response body'}}
+                return {'204': {'description': str(_('No response body'))}}
```

Comment:
no translation for this yet.  As it is likely to be the first & most common string emitted by DRF-spectacular, it would be nice to have this one done.  "Unspecified .." also
But if not in this PR, I'll get it done.

Reply:
i prefer not to ship language files as then we would do a whole lot of languages. if someone cares for those  3-4 strings they can translate them when they do their app anyway.